Formula/boolector.rb

```diff
@@ -0,0 +1,29 @@
+class Boolector < Formula
+  desc "SMT solver for fixed-size bit-vectors"
+  homepage "https://boolector.github.io/"
+  url "https://github.com/Boolector/boolector/archive/refs/tags/3.2.2.tar.gz"
+  sha256 "9a5bdbacf83f2dd81dbed1e1a9f923766807470afa29b73729c947ae769d42b9"
+  license "MIT"
+
+  depends_on "cmake" => :build
+
+  def install
+    system "./contrib/setup-lingeling.sh"
+    system "./contrib/setup-btor2tools.sh"
+    system "./configure.sh", "--prefix", prefix.to_s
```

Comment:
Let's call `cmake` directly instead of using this wrapper script.

Reply:
I've changed it to `cmake`. The code is ready for review.